src/Prefixer.js

```diff
@@ -94,21 +92,21 @@ export function prefix (value, length) {
 				break
 		// display: (flex|inline-flex|inline-box)
 		case 6444:
-			switch (charat(value, strlen(value) - 2 - (~indexof(value, '!important') && 10))) {
+			switch (charat(value, strlen(value) - 3 - (~indexof(value, '!important') && 10))) {
```

Comment:
some changes in this file (like this one) look like they are changing the output, but I don't see that many changes in accompanying tests. Do I interpret this wrongly or should I maybe try to prepare appropriate tests?

Reply:
ok, I think most of the changes are caused by the `;` added to the end of declarations, would love to have a confirmation if that's it or were there some other changes here as well?

Reply:
Yes that's it.